packages/target-ethers-v5/src/codegen/index.ts

```diff
@@ -129,7 +129,8 @@ export function codegenContractFactory(contract: Contract, abi: any, bytecode?:
 
   return `
   import { ${ethersImports.join(', ')} } from "ethers";
-  import { Provider, TransactionRequest } from '@ethersproject/providers';
+  import { Provider } from '@ethersproject/providers';
+  import { TransactionRequest } from "@ethersproject/abstract-provider";
```

Comment:
Looks like `@ethersproject/providers` package uses `TransactionRequest` from `@ethersproject/abstract-provider` itself ([reference](https://github.com/ethers-io/ethers.js/blob/master/packages/providers/src.ts/index.ts#L12)) so shouldn't the existing `TransactionRequest` type already be the correct one?

Reply:
Ah yes you are correct, I just reinstalled `node_modules` and everything works as intended.